mkdocs.yml

```diff
@@ -0,0 +1,107 @@
+# Project information
+site_name: CONNAISSEUR - Verify Container Signatures in Kubernetes
```

Comment:
```suggestion
site_name: CONNAISSEUR - Verify Container Image Signatures in Kubernetes
```

Reply:
yeah, well that is one of those points...I see why one wants to put an 'image' there and I have the same feeling, but it just makes this header long and line breaking and stuff. Without `container` it doesn't work, so I was wondering whether it is not sufficiently correct to stick to how it is

Reply:
Maybe it is something we should discuss and agree upon. Thoughts?

Reply:
Technically it is wrong, but I can see this distinction not being made by most people, so I wouldn't veto it. But with most of the docs using the correct terms by now this still feels inconsistent

Reply:
Maybe I'm mistaken, but I thought we kept the `Image` as the header didnt grow as bad as you thought